docs/api-guide/routers.md

```diff
@@ -72,14 +72,22 @@ Alternatively you can use Django's `include` function, like so…
         url(r'^', include(router.urls)),
     ]
 
-Router URL patterns can also be namespaces.
+To include the router URL patterns with an application namespace pass a
+`(router.urls, app_name)` 2-tuple to `include`.
+
+For example:
 
     urlpatterns = [
         url(r'^forgot-password/$', ForgotPasswordFormView.as_view()),
-        url(r'^api/', include(router.urls, namespace='api')),
+        url(r'^api/', include((router.urls, 'app_name')),
     ]
 
-If using namespacing with hyperlinked serializers you'll also need to ensure that any `view_name` parameters on the serializers correctly reflect the namespace. In the example above you'd need to include a parameter such as `view_name='api:user-detail'` for serializer fields hyperlinked to the user detail view.
+(`include` also allows an optional `namespace` parameter to also create an _instance namespace_.
+This defaults to `None` and will be set to `app_name` if not provided.
+See Django's [URL namespaces docs][url-namespace-docs] for more details.)
```

Comment:
Do we even need to say this? Not sure...

Reply:
Looks better!

Reply:
> and will be set to `app_name` ...

Grrrr. _"...the same as the application namespace..."_ or such.


Don't merge this yet. I'll have a go over the weekend.